HT4_part1.py

```diff
@@ -0,0 +1,59 @@
+# import modules random and string
+import random
+import string
+
+def generate_random_dicts(dict_num_min,

```

Comment:
TO REVIEW:
What if keys/values of dicts will be not consequent? Maybe it is better to pass ranges/lists?